frame/support/src/lib.rs

```diff
@@ -448,6 +448,7 @@ macro_rules! parameter_types_impl_thread_local {
 					}
 
 					/// Mutate the internal value in place.
+					#[allow(unused)]
```

Comment:
Can we mark the whole `impl` as `allow(unused)`? Or even the type itself.

Reply:
Maybe more fine grained control over this is better, that way nothing ever gets lost. What do you think?

Reply:
done